javaparser-core-testing/src/test/java/com/github/javaparser/printer/lexicalpreservation/LexicalPreservingPrinterTest.java

```diff
@@ -1334,4 +1335,56 @@ void commentAddedAtTopLevel() {
         cu.removeComment();
         assertEqualsNoEol("package x;class X{}", LexicalPreservingPrinter.print(cu));
     }
+
+    @Test
+    public void testReplaceStringLiteral() {
+        final JavaParser javaParser = new JavaParser(new ParserConfiguration().setLexicalPreservationEnabled(true));
+
+        final String code = "\"asd\"";
+        final Expression b = javaParser.parseExpression(code).getResult().orElseThrow(AssertionError::new);
+        LexicalPreservingPrinter.setup(b);
```

Comment:
`         final JavaParser javaParser = new JavaParser(new ParserConfiguration().setLexicalPreservationEnabled(true));`

`         LexicalPreservingPrinter.setup(b);`

We're setting up the printer twice here -- once as part of the configuration, and then again afterwards.

Reply:
I'll do this in a follow up PR handling the remaining literal values.